cmd/mattermost/commands/roles.go

```diff
@@ -84,6 +84,8 @@ func makeSystemAdminCmdF(command *cobra.Command, args []string) error {
 			return errUpdate
 		}
 
+		CommandPrintln("Updated", updatedUser.Username, "roles:", updatedUser.Roles)
```

Comment:
Could we have a similar output as `mmctl`?

e.g:
```golang
fmt.Sprintf("%s role is assigned to user %q. Current roles are: %s", role, updatedUser.Username, strings.Join(updatedUser.Roles, ", "))
```

Reply:
Yeah sure, I'll do that :rocket: